bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -179,7 +179,7 @@
                     ButtonFlattener.flatten(this);
                 }});
 
-                add(new JButton(Resources.getIcon("eclipse/download.gif")) {{
+                add(new JButton(Resources.getIcon("eclipse/openbrwsr.gif")) {{
```

Comment:
No, it's just a number of hours and not hours in a day, so, for example, in what Andy showed, he was using 720:00, and the calculation became:

`1000 * (60 * (60 * hh + mm) + ss) = 2592000000`, which is greater than `2^31`, so we were overflowing.